clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -473,6 +475,14 @@ public void recordAssembledValueSize(long bytes, long currentTimeMs) {
     assembledValueSizeSensor.record(bytes, currentTimeMs);
   }
 
+  public void recordAssembledRmdSize(long bytes, long currentTimeMs) {
+    assembledRmdSizeSensor.record(bytes, currentTimeMs);
+  }
+
+  public void recordAssembledRecordSizeRatio(double ratio, long currentTimeMs) {
```

Comment:
Actually, this method is causing confusions:
In `SIT`:
```
public void recordAssembledRecordSizeRatio(double ratio, long currentTimeMs)
```
In `LFSIT`:
```
  protected final void recordAssembledRecordSizeRatio(long size, long currentTimeMs) {
```

Why do we need to implement in this way?
I guess `SIT` can get the max record size and we don't need the override in `LFSIT`, and this method is wrong.

Reply:
Both `SIT` and `LFSIT` have `long size`. The method with `double ratio` should be in `HostLevelIngestionStats`?